nml/actions/action0properties.py

```diff
@@ -605,7 +628,7 @@ def prop23_test(value):
     "retire_early":                 {"size": 1, "num": 0x16},
     "misc_flags":                   {"size": 1, "num": 0x17},
     "refittable_cargo_classes":     [{"size": 2, "num": 0x18}, zero_refit_mask(0x11)],
-    "non_refittable_cargo_classes": [{"size": 2, "num": 0x19}, zero_refit_mask(0x11)],
+    "non_refittable_cargo_classes": refittable_cargo_classes(0x19, 0x25, 0x11),
```

Comment:
Should this be on `refittable_cargo_classes` instead of `non_refittable_cargo_classes`?

Reply:
Is it a method call now `refittable_cargo_classes()`?

Reply:
Maybe you need a picture.

All other cases:

![image](https://github.com/user-attachments/assets/eeba94d0-3c96-4479-96c9-b6de4c2016c5)

This case:

![image](https://github.com/user-attachments/assets/be4d15ed-9311-40c0-989c-e81e87d4f78f)

Reply:
Yes, pictures are best 
<img width="867" alt="image" src="https://github.com/user-attachments/assets/a1e2f4f0-fc8c-4a75-9ecf-54ff1be5c2b2">

Reply:
Okay, so in the case of ships, the `refittable_cargo_classes()` function is used for the `non_refittable_cargo_classes` property.

For trains, road vehicles and aircraft, the `refittable_cargo_classes()` function is used for the `refittable_cargo_classes` property.

Maybe ships are special.

Reply:
Most likely my mistake (it's my code)